### PR TITLE
#29 Fixes issue with electron environment

### PR DESCRIPTION
**Pull request by durgeshahire4**

### CODE REVIEW

File: src/wrappers/start.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 (function (root, factory) {
   'use strict';
-  if (typeof module === 'object') {
+  var isElectron = typeof module == 'object' && process && process.versions && process.versions.electron;
```
Comment (j0k3r) [inline src/wrappers/start.js]: Should be `===` instead of `==`

**Comment by durgeshahire4** [comment]
@j0k3r have a look now....

**Comment by durgeshahire4** [comment]
thanks and good catch

**Comment by j0k3r** [comment]
:+1:

**Pull request merged**